spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/pom.xml

```diff
@@ -18,7 +18,7 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<kafka.version>0.8.2.1</kafka.version>
 		<spring-integration-kafka.version>1.2.0.RELEASE</spring-integration-kafka.version>
-		<rxjava.version>1.0.0</rxjava.version>
+		<rxjava-math.version>1.0.0</rxjava-math.version>
```

Comment:
actually I've changed `rx-java` to `rxjava` since that's how the artifacts are called.